sdcm/sct_config.py

```diff
@@ -2027,8 +2058,8 @@ def verify_configuration(self):
             self._validate_nemesis_can_run_on_non_seed()
             self._validate_number_of_db_nodes_divides_by_az_number()
 
-        if 'extra_network_interface' in self and len(self.region_names) >= 2:
-            raise ValueError("extra_network_interface isn't supported for multi region use cases")
+        # if 'extra_network_interface' in self and len(self.region_names) >= 2:
```

Comment:
why this is commented out ?

Reply:
I do not know why we do not want to add extra network interface in multiDC tests.
I asked Dmitri why we have this restriction and waiting for the answer. So meanwhile I commented out this

Reply:
This is Dmitry's answer:
```
 guess it is related to the problem when SCT blocks traffic to primary interface, while maintaining connection to the node via secondary interface
And probably there a problem with reaching out nodes running on other region
```

Reply:
moved this validation